mne/viz/tests/test_ica.py

```diff
@@ -328,6 +328,22 @@ def test_plot_ica_sources(raw_orig, browser_backend, monkeypatch):
         assert len(fig.mne.ax_hscroll.collections) == 1
     else:
         assert len(fig.mne.regions) == 1
+        assert np.allclose(fig.mne.regions[0].getRegion(), (0.2, 0.3))
```

Comment:
Same here. Then you'll need to import at the top.

I tried to push the change but GitHub wouldn't let me for some reason

Reply:
I can't either.. well that's interesting.